config/config-example.js

```diff
@@ -10,6 +10,17 @@ exports.port = 8000;
 //   know what you are doing.
 exports.proxyip = false;
 
+// cloudenv - automatically configure port and bind address using cloud environment
+//   This should only be enabled if the server is running on a cloud-based host,
+//   such as c9, Heroku, or OpenShift.
+exports.cloudenv = false;
```

Comment:
This should not be here; the entire point of cloudenv is to do this by default instead of needing manual configuration.